website_base_multi_image/static/src/js/website_multi_image.js

```diff
@@ -0,0 +1,154 @@
+/* © 2016 Tecnativa, S.L.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+ */
+odoo.define("website_base_multi_image.images",
+function(require){
+"use strict";
+
+var animation = require("web_editor.snippets.animation");
+var core = require("web.core");
+var Model = require("web.Model");
+var session = require("web.session");
+
+var _t = core._t;
+var $ = require("$");
+
+animation.registry.website_base_multi_image =
+animation.Class.extend({
+    selector: ".js_image_container",
+    start: function (editable_mode) {
+        if (editable_mode) {
+            return;
+        };
+        this.$image_preview_thumb = this.$(".thumbnail-images");
+        this.$file_input = this.$("#file_input");
+        this.$label_input = this.$(".image_upload");
+        this.$file_input_multi = this.$("#file_input_multi");
+        this.$carousel_container = this.$(".carousel-inner")
+        this.image_added_id = 0;
+        core.qweb.add_template(
+            "/website_base_multi_image/static" +
+            "/src/xml/website_multi_image.xml");
+        this.bind_events();
+    },
+    bind_events: function () {
+        var this_ = this;
+        this.$el.on("change", "#file_input", function (event) {
+            return this_.image_preview(event);
+        });
+        this.$el.on("click", ".remove-image", function (event) {
```

Comment:
You should use proxy here `this.$el.on("click", ".remove-image", $.proxy(this.image_remove, this));`